cli/Cargo.toml

```diff
@@ -1,14 +1,15 @@
 [package]
 name = "subxt-cli"
 version = "0.27.1"
```

Comment:
I think for now that'd probably be the way to go, although I acknowledge that if we want to patch-bump something on it's own it would not work so well. (I think though that we will endeavour to keep versions in sync for simplicity for the time being though, as we have done so far)